lib/ui/window/window.cc

```diff
@@ -261,13 +261,21 @@ void Window::UpdateAccessibilityFeatures(int32_t values) {
 
 void Window::DispatchPlatformMessage(fml::RefPtr<PlatformMessage> message) {
   std::shared_ptr<tonic::DartState> dart_state = library_.dart_state().lock();
-  if (!dart_state)
+  if (!dart_state) {
+    FML_DLOG(WARNING)
+        << "Dropping platform message for lack of DartState on channel:"
```

Comment:
nit: `on channel: "`

Should this be a DLOG or a LOG? Probably fine to be a DLOG for now, but it will only show up in unopt builds.

Reply:
Talked offline, DLOG is good for now.